### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by KingsleyIbe**
# Group project - changes

In this project, we built a Space travelers' website using the Space X API.

Step by step implementation can be found [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md)

### CODE REVIEW

File: src/pages/RocketsCards.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                onClick={reserveRockets}
+                id={rocket.id}
+              >
+                Reserve Rocket
+              </button>
+            )}
+          </div>
+        </div>
+      ))}
+    </div>
```
Comment (bimbolabuari) [inline src/pages/RocketsCards.js:77]: As required, Kindly follow the design guide including the spacing and color 🙂 
> Your Rocket page design
image

> Requirement design wireframe
image

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        className="nav-item"
+        to="/myProfile"
+      >
+        {' '}
+        My Profile
+        {' '}
+      </NavLink>
+    </nav>
+
+  </header>
```
Comment (bimbolabuari) [inline src/components/Header.js:40]: Let's follow the required design wireframe 👍 

> Your webpage header
image
 
> Required design header
image

File: src/components/Profile.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ProfileRocketsDisplay from '../pages/ProfileRocketsDisplay';
+
+const Profile = () => (
+  <div>
+    <h1>My Rockets</h1>
+    <ProfileRocketsDisplay />
+  </div>
+);
```
Comment (bimbolabuari) [inline src/components/Profile.js:9]: As required, The My Profile section should display all reserved rockets and space missions, Its currently blank after I reserved all missions and rocket, Kindly fix this 🙂 

image

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi Team, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Correct Gitflow
- No linter errors
- You documented your work in a professional way.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
- [ ] Kindly write unit tests with React Testing Library and Jest

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Comment by KingsleyIbe** [comment]
Hey @bimbolabuari 

Thank you for your feedback. Changes will be implemented.

File: src/pages/RocketsCards.js (hunk 1 above)
Comment (benjp009) [reply src/pages/RocketsCards.js]: That has been changed

File: src/components/Profile.js (hunk 3 above)
Comment (benjp009) [reply src/components/Profile.js]: changed

File: src/components/Header.js (hunk 2 above)
Comment (benjp009) [reply src/components/Header.js]: changed

**Review by ZeenatLawal — APPROVED** [review]
Hi @KingsleyIbe and @benjp009, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by KingsleyIbe** [comment]
Hey @ZeenatLawal 

Thanks for approving our project.

**Pull request merged**